### DISCUSSION TITLE
Can not create a team role if global role with the same name exists

### DISCUSSION THREAD

**Discussion by sohailahmad07** [spatie/laravel-permission#2671] [Q&A] [answered]
HI i dont know if this is  bug or not so that why i am asking the question first here 

i have an app and i have enabled teams' permission but i have the same roles globally and for team for example i have a global role "admin", and a team also have a role of an admin the problem is if create a global role admin first the can not create a team roles with admin 

However as long as i don't have a global roles admin i can create the admin roles for many teams as there a way to create a role with the same name for both global and teams

here a gif explanation you will see when i create a global role then i can not create a team role with the same name

![Screen Capture_select-area_20240523135747](https://github.com/spatie/laravel-permission/assets/58299055/45b13e5f-66d8-4d69-900c-450630cbace4)

**Comment by erikn69** [comment, marked as answer]
Its a expected behaivor, if global role  exists, new role with the same name is a duplicated role, 
There cannot be two roles with the same name.
